regulations/templates/regulations/comment.html

```diff
@@ -2,9 +2,14 @@
 
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+
 </head>
 
 <body>
+  {% if mark_as_draft %}
+    <div class="watermark">
+  {% endif %}
```

Comment:
Super duper minor, but if you need to make a change to fix conflicts, would this work?
```html
<body {% if mark_as_draft %}class="watermark"{% endif %}>
```